### ISSUE TITLE
Engineering armory has no button.

### ISSUE THREAD

**Issue by troytroy400** [RMC-14/RMC-14#4326]
## Description
The upper engineering armory doors has no buttons, engineering is unable to access their armor vests. 

**Reproduction**
N/A

**Screenshots**
N/A

**Additional context**
Relevant to - #2393

**Comment by Vermidia** [comment]
Is this fixed?

**Comment by troytroy400** [comment]
> Is this fixed?

No, but its also not parity. It SHOULD be tied to the red alert.

**Comment by Dutch-VanDerLinde** [comment]
> > Is this fixed?
> 
> No, but its also not parity. It SHOULD be tied to the red alert.

Engi armory is now tied to red alert

**Comment by Vermidia** [comment]
Fixed enough then.
